app/validation/domain.py

```diff
@@ -128,11 +128,10 @@ def validate_non_zero_field_length(self, field_name, length, max_field_len):
         self.validate_field_length(field_name, length, max_field_len)
 
     @staticmethod
-    def validate_field_length(field_name, length, max_field_len):
+    def validate_field_length(field_name, length, max_field_len, data=None):
         if length > max_field_len:
             logger.debug("{0} field is too large {1}  max size: {2}".format(field_name, length, max_field_len))
-            raise ValidationError('{0} field length must not be greater than {1}.'.format(field_name, max_field_len))
-
+            raise ValidationError('{0} field length must not be greater than {1}.'.format(field_name, max_field_len), field_name, [], data)
 
 class DraftSchema(Schema):
```

Comment:
Since DraftSchema is almost the same as the MessageSchema could we just make it a subclass of MessageSchema and overwrite the different functions?